.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,6 +22,13 @@ $
 PROPOSAL:
 
 
+<!--- 
+If you want to trigger adhoc build of hybrid app from specific Mobile-Expensify branch or commit please specify it like follows:
+
+MOBILE-EXPENSIFY: branch or commit
```

Comment:
so if its from a fork, we should use commit? Or will branch name from swm fork work?

Reply:
Well, it's good question. I have to check if it will work for sure if Mobile-Expensify is a fork

Reply:
Thanks @mountiny! It was good catch. I've updated the PR to look for PR number instead of commit/branch that exist only in fork. Now the checkout is performed on Mobile-Expensify PR ref.